brianlong/www.validators.app
# 183873878 - remove ftx

#### What's this PR do?
Remove ftx logic and ftx from the views.

#### How should this be manually tested?
- `rails test`
- verify sol prices page
- execute `rails r script/one_time_scripts/remove_ftx_sol_prices.rb` and check if there are any FTX sol prices records

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183873878)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
